src/api/app/views/webui/project/new_package_branch.html.erb

```diff
@@ -21,7 +21,7 @@ package.</p>
   <% end %>
   </ul>
 <% end %>
-<%= form_tag controller: :package, action: :branch do %>
+<%= form_tag controller: :package, action: :branch, params: { add_repositories: 1 } do %>
```

Comment:
Is it valid to have this always enabled? I thought it's only for branching remote packages